content/api/entity-reference/volunteer/index.html

```diff
@@ -3,7 +3,7 @@
 name: Volunteer Entity Reference
 title: Volunteer Entity Reference
 order: 90
-published: false
+published: true
```

Comment:
If we are turning on the volunteer entity reference, do we need a volunteer changelog to announce the change? Seems like we've announced new entities in the past instead of just publishing the reference.